filechanges/opt/musicbox/setsound.sh

```diff
@@ -179,33 +179,56 @@ then
 cat << EOF > /etc/asound.conf
 pcm.!default {
     type plug
+    slave.pcm plugequal;
+}
+pcm.plugequal {
+    type dmix
+    ipc_key 1024
+    controls "/home/mopidy/.alsaequal.bin"
```

Comment:
Ok Yeh I figured it was to store the state. And then I remembered that
/home/mopidy does exist in 0.6. But it won't exist in 0.7 since we'll be
using mopidy from apt which sets up the mopidy user account properly. Since
this is system state, it probably deserves to go somewhere system-ish
anyway.
On 25 Jul 2015 09:37, "John Cass" <notifications@github.com> wrote:

> In filechanges/opt/musicbox/setsound.sh
> <https://github.com/woutervanwijk/Pi-MusicBox/pull/307#discussion_r35481120>
> :
>
> > @@ -179,33 +179,56 @@ then
> >  cat << EOF > /etc/asound.conf
> >  pcm.!default {
> >      type plug
> > +    slave.pcm plugequal;
> > +}
> > +pcm.plugequal {
> > +    type dmix
> > +    ipc_key 1024
> > +    controls "/home/mopidy/.alsaequal.bin"
>
> Here's some sample output of alsamixer -D equal, but you'll have to run sudo
> apt-get install -y libasound2-plugin-equal first.
>
> [image: mopidy ssh 180x46]
> <https://cloud.githubusercontent.com/assets/10268911/8888708/e444550a-32b8-11e5-98c4-3c3dae26d8a3.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/woutervanwijk/Pi-MusicBox/pull/307/files#r35481120>.
>